### ISSUE TITLE
9term: stuck on resize event in X11 when mouse is at window border

### ISSUE THREAD

**Issue by gdiazlo** [9fans/plan9port#465]
Hello

While using 9term under some window managers on Linux (xfce for example), trying to resize the window makes it stuck on resize event even if you're not dragging anymore (while the mouse pointer is at the window border). Acme behaves fine.

Did this happen to anyone?

The code in wind.c on both acme and 9term are too different between them to appreciate what have caused this without being familiar with it.

**Comment by Pipeline79** [comment]
Yes I'm having this annoying problem too.  I'm using xfce-ubuntu.  I just tried asking about it the google dev group but the message hasn't appeared yet for some reason, don't know if messages are pre moderated or something.
